lars-reimann/Safe-DS
# feat: remove_outliers() method for table class

Closes #308.

### Summary of Changes

Adds Table.remove_outliers() method that removes every row that is more then 3 standard deviations removed from the average. Also adds Table.list_columns_with_numerical_values(), which is used in Table.remove_outliers(). Also tests.

### Testing Instructions

Create Table, call Table.remove_outliers(), check if expected rows have been removed.
